WooCommerce/Classes/POS/Analytics/POSCollectOrderPaymentAnalytics.swift

```diff
@@ -35,6 +57,11 @@ final class POSCollectOrderPaymentAnalytics: POSCollectOrderPaymentAnalyticsTrac
         let elapsedTimeSinceCardTapped = calculateElapsedTimeInMilliseconds(since: cardReaderTapped)
 
         analytics.track(event: .PointOfSale.cardPresentCollectPaymentSuccess(
+            forGatewayID: paymentGatewayAccount?.gatewayID,
+            countryCode: configuration.countryCode,
+            paymentMethod: capturedPaymentData.paymentMethod,
+            cardReaderModel: connectedReaderModel,
+            siteID: siteID,
```

Comment:
Question: Do we need to track `site_id` explicitly? We already capture `blog_id` as part of the event.